lib/java/com/google/android/material/appbar/res/values-v23/styles.xml

```diff
@@ -15,6 +15,29 @@
   ~ limitations under the License.
 -->
 <resources>
+  <style name="ThemeOverlay.MaterialComponents.Toolbar.Surface" parent="">
```

Comment:
Let's follow the standard Base.ThemeOverlay -> ThemeOverlay pattern for these theme overlays to avoid duplicate attributes.